README.md

```diff
@@ -1 +1,48 @@
-# Mobile-Portfolio
\ No newline at end of file
+# Mobile-Portfolio![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Mobile-Portfolio.
+
+> This project is a portfolio.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+An editor : VS code.
+
+### Setup
+Clone this repository by using the command line : 
+`git clone https://https://github.com/Rikol137/Mobile-Portfolio/`
+
+## Authors
+
+👤 **Kelvin Kiplimo**
+
+- GitHub: [@Rikol137](https://https://github.com/Rikol137)
+- Twitter: [@RikolK](https://twitter.com/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/KelvinKimwetich/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
[OPTIONAL] Kindly have this pointing to the issues page of this project. Currently it opens an empty link with a 404 error.